sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -866,7 +896,29 @@ func (host *nodeLanguageHost) InstallDependencies(
 func (host *nodeLanguageHost) RuntimeOptionsPrompts(ctx context.Context,
 	req *pulumirpc.RuntimeOptionsRequest,
 ) (*pulumirpc.RuntimeOptionsResponse, error) {
-	return &pulumirpc.RuntimeOptionsResponse{}, nil
+	var prompts []*pulumirpc.RuntimeOptionPrompt
+	rawOpts := req.Info.Options.AsMap()
+
+	if _, hasPackagemanager := rawOpts["packagemanager"]; !hasPackagemanager {
+		prompts = append(prompts, &pulumirpc.RuntimeOptionPrompt{
+			Key:         "packagemanager",
+			Description: "The packagemangager to use for installing dependencies",
+			PromptType:  pulumirpc.RuntimeOptionPrompt_STRING,
+			Choices: []*pulumirpc.RuntimeOptionPrompt_RuntimeOptionValue{
```

Comment:
We're not offering `auto` as a choice here. That feels right to me, but I am not 100% sure about it.